### PR TITLE
[NO GUARD - DO NOT MERGE] LinkedIn Normalization (Backend)

### PR DESCRIPTION
**Pull request by leogrim**
Modifies the normalization service to handle LinkedIn profiles
@yingjieMiao @eishay @ymatsuda

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/normalizer/NormalizationService.scala [hunk 1]
```diff
@@ -181,6 +213,23 @@ class NormalizationServiceImpl @Inject() (
...
+        def parse(doc: Document): String = doc.getElementsContainingText(s"newTrkInfo = '${privateProfileId},' + document.referrer.substr(0,128)").text()
+      }
+
+      for { publicProfileOption <- scraperPlugin.scrapeBasicArticle(publicProfileCandidate.url, Some(idExtractor)) } yield publicProfileOption match {
+        case Some(article) => article.content.nonEmpty
+        case None => false
+
+      }
+    }
+    def persistFailedContentChecks() = {}
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/normalizer/NormalizationService.scala]: function not defined?

**Comment by yingjieMiao** [comment]
it may be helpful to factor out those private classes and objects. You can make them private at the package level. I find they are mixed with some helper functions, which makes the code less readable.

**Comment by leogrim** [comment]
Ok !

2013/8/28 yingjieMiao notifications@github.com

**Comment by yingjieMiao** [comment]
looks good on the high level. Let's test it later.

**Pull request merged**